api/jobs.yaml

```diff
@@ -213,6 +213,9 @@ definitions:
       attempts:
         type: integer
         description: Number of times the task was run
+      subworkflowId:
```

Comment:
There is no concept of a "workflow" in the spec currently. This should refer to a job instead... jobId, subJobId, childJobId? Something like that?

Note: jobId is already defined above to refer to the internal backend job ID - I wouldn't mind renaming that one to something like executionId though.

Reply:
Good point! I agree with changing `jobId` to `executionId` since "jobs" mean something different in this context. And after some discussion with @dshiga, renaming`subworkflowId` to `jobId` would make sense here since it's the id of the job that corresponds to this task.